src/components/SideBarAccordion.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+
+import Accordion from 'cozy-ui/transpiled/react/Accordion'
+import AccordionDetails from 'cozy-ui/transpiled/react/AccordionDetails'
+import AccordionSummary from 'cozy-ui/transpiled/react/AccordionSummary'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import BottomIcon from 'cozy-ui/transpiled/react/Icons/Bottom'
+import { makeStyles } from 'cozy-ui/transpiled/react/styles'
+
+const useStyles = makeStyles({
+  accordion: {
+    boxShadow: 'none',
+    border: 'none',
+    backgroundColor: 'transparent',
+    marginTop: '0 !important'
+  },
+  summary: {
+    textTransform: 'none',
+    color: 'var(--coolGrey)',
```

Comment:
Is it still recommended to use this color @JF-Cozy ?

Reply:
nope

Reply:
Then, what color should I use ?